src/_posts/platform/app/2000-01-01-crash.md

```diff
@@ -1,69 +1,217 @@
+requirements.
 
-You'll also find the 'crash' events in your app timeline on the dashboard.
+If your application doesn't need a `web` or `tcp` process type, make sure to
+[scale the unnecessary process type to zero]({% post_url platform/app/2000-01-01-web-less-app %}#deploy-a-web-less-application).
 
-## What to do?
+### Understanding Hook Errors
 
-We strongly advise you to look at the logs of your application using the web
-dashboard or by using the [CLI tool](http://cli.scalingo.com).
+If your application has a [`postdeploy` process type]({% post_url platform/app/2000-01-01-procfile %}#special-process-types),
+the platform can throw a **Hook Error** if the post-deployment process fails.
 
-According to the information gathered, you should then modify your application to
-fix the issues which lead to this instability.
+- In such a case, the deployment fails with a **hook-error** status.
+- If your application is already running, its code isn't updated and the former
+  version keeps running.
 
 {% note %}
-  After a successful deployment, a manual restart, or scaling of your application, we
-  cancel any queued restart job, and the cool down time after the next crash is resetted.
+  Hook Errors can only occur if you have a `postdeploy` process type.
 {% endnote %}
 
-## Need support?
+#### Fixing Hook Errors
+
+Hook Errors are generally caused by an error in your codebase or by some
+misconfiguration. To recover from it, we first advise to investigate the logs
+of your failed deployment to understand the root cause. After fixing it,
+trigger a new deployment by pushing your updated code to Scalingo.
+
+
+## Understanding Runtime Errors
 
-Don't hesitate to contact us at [support@scalingo.com](mailto:support@scalingo.com).
+**Runtime Errors** are errors that happen during the execution of the process.
+Per definition, they can only occur when your container has reached its
+*running* state (which means your deployment is considered successful by the
+platform).
+
+Unfortunately, and even if it has reached this state, unexpected errors can
+still lead your application to crash.
+
+The most common causes are:
+
+- Incompatible dependencies
+- Segfault of a library/runtime
+- Configuration issues
+- Bugs in your application code
+- Uncaught exception in your code (especially with non-compiled languages)
+- Unsufficient resources
+- Temporary error/unavailability of an external resource
+
+A Runtime Error can have several consequences, depending on the severity of the
+error and the impact it has on your application:
+
+- Reduced Performances: slow response time typically result in poor user
+  experience.
+- Downtime: the unavailability of your application, even if temporary, can have
+  a significant impact on business operations.
+- Data loss, or Corrupt Data: losing strategic data can have serious
+  consequences for your organization or for your users/customers, especially in
+  an HDS context.
```

Comment:
suggestion: add a link to the HDS page (https://doc.scalingo.com/compliance/hds)?